db/schema.rb

```diff
@@ -125,7 +125,6 @@
     t.boolean "anyone_pay_child_support_alimony_arrears"
     t.boolean "anyone_pay_student_loan_interest"
     t.boolean "anyone_self_employed", default: false
-    t.datetime "birthday"
```

Comment:
does this mean we aren't storing date of birth anymore?

Reply:
we are storing in on each member. we used to store it here but no longer do so!

Reply:
answered my own question. Looks like it's stored on member.